### ISSUE TITLE
Allow more substitutions for keyboard-limited characters

### ISSUE THREAD

**Issue by Sancoraa** [entertrained/entertrained-issues#16]
Characters like "ú" are considered equivalent to a standard "u".
In French, sentences starting with "É" or "À" are problematic to type on standard AZERTY keyboards. Substituting with plain "E" or "A" should be acceptable.
There is also the character "œ" that has the same issue, but I cannot think of a way to fix this.

**Comment by entertrained** [comment]
Hi Sancoraa,

If you're talking about imported French books, then as a solution (at least temporary) I would suggest to edit the book and change the language to English (all it does is specifies which set of characters to use), then all characters that are not a part of standard Latin alphabet can be typed with any key.

**Comment by caderek** [comment]
Here's the set of characters that the app currently requires to type if the Book's language is set to French:

```txt

Letters:
    
AaÀàÂâÆæBbCcÇçDdEeÉéÈèÊêËëFfGgHhIiÎîÏïJjKkLlMmNnOoÔôŒœPpQqRrSsTtUuÙùÛûÜüVvWwXxYyŸÿZz

Symbols:

`1234567890-=~!@#$%^&*()_+[]\{}|;':",./<>? 
```

As I'm not French, I would be grateful if you, or some other native speaker, could mark the exact list of characters that are usually not used on the standard AZERTY French keyboard layout.

**Comment by Sancoraa** [comment]
From the list you sent me, here are the characters that you can't write on an AZERTY keyboard :

`ÀÆæÇÉÈŒœÙ`

**Comment by caderek** [comment]
Thank you, I added the fix in the version 1.15.2 (you may need to refresh the page to see the changes, you can find the version number in the footer).
